html/clases.html

```diff
@@ -0,0 +1,66 @@
+<html>
+<head>
+<style>
+.city {
+  background-color: tomato;
+  color: white;
+  border: 2px solid black;
+  margin: 20px;
+  padding: 20px;
+}
+</style>
+</head>
+<body>
+
+<div class="city">
+<h2>London</h2>
+<p>London is the capital of England.</p>
+</div> 
+
+<div class="city">
+<h2>Paris</h2>
+<p>Paris is the capital of France.</p>
+</div>
+
+<div class="city">
+<h2>Tokyo</h2>
+<p>Tokyo is the capital of Japan.</p>
+</div>
+
+<style>
+    .note {
+      font-size: 120%;
+      color: red;
+    }
+    </style>
+    </head>
+    <body>
+    
+    <h1>My <span class="note">Important</span> Heading</h1>
+    <p>This is some <span class="note">important</span> text.</p>
+
+    <style>
+        .city {
+          background-color: tomato;
+          color: white;
+          padding: 10px;
+        }
+        </style>
+        </head>
+        <body>
```

Comment:
You have multiple <body> task, in one html file you should have jsut one <body> one <head> and one <html> tag